### DISCUSSION TITLE
NVIDIA GPU Power Draw in Optimized Mode

### DISCUSSION THREAD

**Discussion by frknkcr** [seerge/g-helper#3123] [Q&A] [answered]
I'm using my laptop in optimized mode, and I've noticed that when the external NVIDIA GPU is not being used, it doesn't completely shut down. Instead, it intermittently turns on and consumes power in short bursts. This happens even though the GPU is set to Optimus mode.

Why isn't the GPU fully disabled in this case, and is there a way to prevent it from drawing power when it's not needed?

![image](https://github.com/user-attachments/assets/ac184484-4246-4781-b673-815588c09758)

**Comment by seerge** [comment, marked as answer]
@frknkcr hello,

The one and only thing `Optimized` mode does is sets `Eco` on battery (disables dGPU completely) and `Standard` when plugged in (enables GPU back). So when enabled dGPU will obviously use power.

As for `Optimus` / `NVidia GPU only` toggle in Nvidia Control Panel - it only switches if iGPU or dGPU should drive your internal display. Even in `Optimus` dGPU is still available and can be potentially used by windows or apps.
